getting_started/setup_linux/setup_ktest_environment.rst

```diff
@@ -331,50 +333,51 @@ Save the following in sof-dev.conf.
   BUILD_NOCLEAN = 1
 
 
-For Fedora and derived distributions, make the following changes:
+For targets running Fedora and derived distributions, make the following changes:
 
 .. code-block:: perl
 
-  GRUB_MENU    = "title" of the kernel entry as displayed by: 'grubby --info=ALL'
+  GRUB_MENU    = ${LOCALVERSION}
   GRUB_REBOOT  = grub2-reboot
   REBOOT_TYPE  = grub2bls
-  POST_INSTALL = ssh  -o 'ProxyCommand none' $SSH_USER@$MACHINE sudo dracut --hostonly --force --kver ${LOCALVERSION}
+  POST_INSTALL = ssh  -o 'ProxyCommand none' $SSH_USER@$MACHINE sudo dracut --hostonly --force /boot/initramfs-${LOCALVERSION}.img $KERNEL_VERSION
 
-7. Build and test
+4. Build and test
 -----------------
 
 .. code-block:: bash
 
+   # This can take a while, so don't kill it if it appears to freeze
    ./ktest.pl sof-dev.conf
 
 If this does not work, make sure you have all the following files in the
 local directory:
 
 * ktest.pl
 * sof-dev-cat
-* sof-dev
+* linux
 * sof-dev-build
 * sof-dev.conf
 * sof-dev-defconfig
 
 Ktest will compile, install the new kernel, and reboot. Prompt
-detection only works with a UART, not over SSH, so you will have to
-``Control-C`` manually when the console is not enabled.
+detection only works with a UART, not over SSH, so you will get an error if the console is not enabled. In this case, you'll have to
+``Control-C`` manually. Even though the script reports an error, the new kernel should still be installed and your target device should boot it. Check which kernel is booted by running ``uname -r``.
```

Comment:
People who have no prompt detection should simply use `TEST_TYPE = install` instead of hitting Ctrl-C after an undefined amount of time.

Reply:
Should I just set that as the default since most people won't have that?

Reply:
Makes sense to me.